tests/csm/rest/test_csm_users.py

```diff
@@ -1981,66 +1984,6 @@ def test_6220(self):
         self.log.info(
             "Step 2: Verified that csm manage user is not able to create duplicate csm user")
```

Comment:
U need to remove this line, Can u create new test with below steps and write in backlog sheet.

Reply:
will create a new test. Why does this line need to be removed?